src/index.js

```diff
@@ -18,34 +18,47 @@ function resolve (dir) {
 
 /**
  * Create a webpack config usable for demosplan-core addons.
- * 
+ *
  * ## Usage
- * 
+ *
  * ```js
  * const DemosplanAddon = require('@demos-europe/demosplan-addon')
- * 
+ *
  * module.exports = DemosplanAddon.build(
- *   'my-addon-name', 
+ *   'my-addon-name',
  *   { 'MyAddon': DemosPlanAddon.resolve('src/index.js') }
  * )
  * ```
- * 
+ *
  * @param {String} addon_name the name, d'uh
- * @param {Object} entrypoints name-mapped entry points dictionary 
+ * @param {Object} entrypoints name-mapped entry points dictionary
  * @returns {Options} webpack configuration
  */
 function configBuilder(addon_name, entrypoints) {
   const isProduction = process.env.NODE_ENV == 'production'
 
+  /**
+   * Transform the entry points object into a usable format
+   * so a browser is able to digest the output but still keep
+   * the simple webpack config mentioned above to create addons
+   */
+  for (const key of Object.keys(entrypoints)) {
+    entrypoints[key] = {
+      import: entrypoints[key]
+    }
+    entrypoints[key]['library'] = {
+      name: key,
+      type: 'window'
```

Comment:
Maybe first-time readers of these lines will get a jump-start when mentioning that you're utilizing the EntryDescription object alongside configuring library options per entry point, as seen in the [Webpack documentation](https://webpack.js.org/configuration/output/#outputlibrary), within a short comment.

Reply:
Added a more verbose comment here: https://github.com/demos-europe/demosplan-js-addon/pull/14/commits/4163c2a6946ddb6b09f8928c48dc0e210af22347